Dockerfile.eudico

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.18.3-buster
```

Comment:
I don't get this image, what do we use it for?

Reply:
I was using this as the base image for vscode remote docker development. I will add more commands to it when `eudico-stats` is ready. The plan is to replace the Dockerfile.lotus for eudico.

Reply:
Let's not replace it for now, but let's have a Dockerfile.lotus and Dockerfile.eudico. We are still a fork, and we want to be able to pull potential changes they make upstream. 

Let's have a minimal dockerfile working in this PR for Eudico or let's remove it for now (this is a raw Go image without much).

Reply:
I'll remove for now.